webpack.config.js

```diff
@@ -352,6 +353,9 @@ function getWebpackConfig( { cssFilename, externalizeWordPressPackages = false }
 				} ),
 			shouldEmitStats && new webpack.ProgressPlugin( createProgressHandler() ),
 			new BuildCustomPropertiesCssPlugin(),
+			process.env.STRIP_OLD_TIMEZONE_DATA
+				? new MomentTimezoneDataPlugin( { startYear: 2000 } )
+				: false,
```

Comment:
@dmsnell In your opinion, should I remove the env var entirely and go back to what I had, or should I replace it with a negative one that can be turned on if needed, e.g. `KEEP_OLD_TIMEZONE_DATA`?

Reply:
My thought was to go back to the original until we have a real need for the older data. Right now it’s not clear we ever will and so I’m thinking this could be a case of needless abstraction. Obviously it’s not a big deal here as a single conditional, but why bother if we have no need 🤷‍♂️

Reply:
Sure, that works for me! 👍

@blowery: does that sound ok?

Reply:
yup, sounds great!

Reply:
Sweet, thank you both! Removed the last commit, going back to the original changes (though rebased on current master).